### PR TITLE
README emojis contribute links

### PR DESCRIPTION
**Pull request by KoenBal**
### Description
As part of the project README redesign
added Emojis
added links
added contribution options

Rationale: For some users the celo monorepo could be the first impression they have of celo. A 
A nice README is a good way to help people engage in the project as well. A project with nice README and screenshots will get the attention of users better since it’s a direct way to explain why this project matters, and why people should use and contribute to the project. Good README should also include enough details to help a new user get started, e.g. how to compile, how to install, and how to start integrating.

README educational content I used.
[A beginners guide to writing kickass readmes](https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3)
[AWESOME READMEs](https://github.com/matiassingers/awesome-readme)

### Tested
tested links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,28 +15,61 @@ Celo is oriented around providing the simplest possible experience for end users
...
 See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a list of active or proposed tasks. Feel free to create new issues to report bugs and/or request features. Please add labels to your issues, tagging the appropriate package/area.
 
-## License & Contributing
 
-All packages are licensed under the terms of the Apache 2.0 License unless otherwise specified in the LICENSE file at package's root.
+### ✍️ Contributing
+
+Feel free to jump on the celo 🚂🚋🚋🚋.  Improvements and contributions are highly encouraged! 🙏👊
+
+See the [contributing guide](https://github.com/celo-org/celo-monorepo/tree/master/.github/CONTRIBUTING.md) for details on how to participate.
```
Comment (timmoreton) [inline README.md]: you can use a relative link `CONTRIBUTING.md` here

File: README.md [hunk 2]
```diff
@@ -15,28 +15,61 @@ Celo is oriented around providing the simplest possible experience for end users
...
+[![GitHub issues by-label](https://img.shields.io/github/issues/celo-org/celo-monorepo/1%20hour%20tasks)](https://github.com/celo-org/celo-monorepo/issues?q=is%3Aopen+is%3Aissue+label%3A%221+hour+tasks%22)
+
+Not yet ready to contribute but do like the project? Support celo with a ⭐ or share the love in a [![Twitter URL](https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Fcelo.org%2F)](https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DkKggE5OvyhE&via=celohq&text=Checkout%20celo%21%20Love%20what%20they%20are%20building.&hashtags=celo)
```
Comment (timmoreton) [inline README.md]: nit: celo -> Celo

**Review by timmoreton — APPROVED** [review]
minor comments you might like to look at - otherwise I'll merge 👍

**Comment by KoenBal** [comment]
@timmoreton Did the fixes, Thx for the review.

**Pull request merged**